Editor/Credentials/Strings.cs

```diff
@@ -130,5 +130,11 @@ public static class Strings
         public const string SettingsUIDeployNextStepLabel = "SettingsUIDeployNextStepLabel";
         public const string LabelOpenSdkIntegrationDoc = "LabelOpenSdkIntegrationDoc";
         public const string LabelOpenSdkApiDoc = "LabelOpenSdkApiDoc";
+        
+        public const string TabLanding = "TabLanding";
+        public const string TabCredentials = "TabCredentials";
+        public const string TabAnywhere = "TabAnywhere";
+        public const string TabEC2 = "TabEC2";
```

Comment:
nit: maybe `TabManagedEC2` for 1:1 mapping with what UX looks like?

Reply:
Name updated